pkg/provider/vault/vault.go

```diff
@@ -450,10 +451,13 @@ func (v *client) PushSecret(ctx context.Context, value []byte, remoteRef esv1bet
 			"managed-by": "external-secrets",
 		},
 	}
+	secretVal := make(map[string]interface{})
+	err := json.Unmarshal(value, &secretVal)
+	if err != nil {
+		return fmt.Errorf("failed to convert value to a 'map[string]interface{}': %w", err)
```

Comment:
I think it would be helpful for the user if we point out that we expect json here.